### ISSUE TITLE
Add support for discussions API

### ISSUE THREAD

**Issue by Maxooo** [GitLabPHP/Client#347]
It seems that the `discussions` API is not supported yet (https://docs.gitlab.com/ce/api/discussions.html). Are there any plans to add this API or should I create a PR for this ?

Thank you for the time spent and the support given for this library.

**Comment by otrelin** [comment]
Also interested in this question - are you planning to add Discussions API?

**Comment by otrelin** [comment]
@m1guelpf I've already done https://github.com/m4tthumphrey/php-gitlab-api/pull/378
When will be the new release with these changes?

I also suggest in `lib/Gitlab/Api/Issues.php` add methods `showNotes()`, `showNote()` etc. instead of `showComments()`, `showComment()` etc. respectively (as [here](https://github.com/m4tthumphrey/php-gitlab-api/blob/master/lib/Gitlab/Api/MergeRequests.php#L198)). Or should I make new PR for it?

**Comment by m1guelpf** [comment]
@Maxooo Just released a new version 🎉

**Comment by glensc** [comment]
#378 does not seem to be complete, for example whole concept of commit is missing:
- https://docs.gitlab.com/ee/api/discussions.html#get-single-commit-discussion-item

**Comment by glensc** [comment]
added something quickly for specific use-case:
- https://github.com/m4tthumphrey/php-gitlab-api/pull/501

**Comment by glensc** [comment]
@GrahamCampbell I think this can be closed, as it takes more effort to find what's missing that just wait for someone to request what they are missing. and reaching 100% compatible is probably impossible considering they make changes to api with each release.

when I worked with https://github.com/GitLabPHP/Client/pull/501 for https://github.com/glensc/slack-unfurl-gitlab/pull/10 I found there are many duplicate endpoints that return same data, with some variation of input, i.e from the perspective of the commit, from the perspective of a discussion, from the perspective of comments. yes, the terminology is also confusing as they seem to be the same thing (discussion, comments, notes), but really are not. and I'm not sure which way gitlab is heading.
